test/functional/sqlsrv/TC64_TransactionProc.phpt

```diff
@@ -31,79 +31,71 @@ function Transaction($minType, $maxType)
                 $data = null;
                 break;
             default:
-                $data = GetSampleData($k);
+                $data = getSampleData($k);
                 break;
         }
         if ($data != null) {
-            $sqlType = GetSqlType($k);
+            $sqlType = getSqlType($k);
+            $driverType = getDriverType($k);
```

Comment:
the name of this function is very misleading. I thought getDriverType gets the PHP zend type... can you change the name of this to getSqlsrvSqltype() or something more clear?

Reply:
I agree that it's confusing, but that was one of the common functions originally in MsCommon... I'll see how many tests are affected by the renaming.